src/runtime/entitlements-manager-test.js

```diff
@@ -1937,6 +1941,55 @@ describes.realWin('EntitlementsManager', (env) => {
 
       await manager.getEntitlements();
     });
+
+    it('should return null promise when fetching interventions without article', async () => {
+      manager = new EntitlementsManager(
+        win,
+        pageConfig,
+        fetcher,
+        deps,
+        /* useArticleEndpoint */ false
+      );
+      sandbox.stub(manager, 'getArticle').resolves(null);
+
+      expect(await manager.getAvailableInterventions()).to.equal(
+        null,
+        'getAvailableInterventions should return null'
```

Comment:
Could we log an error that advises a dev on how to add support for this API?

Reply:
Done.